src/rootlesscontainers/rootlesscontainers.pb-c.c

```diff
@@ -0,0 +1,103 @@
+/* Generated by the protocol buffer compiler.  DO NOT EDIT! */
```

Comment:
Auto-generated files should not be included in the repository, (See: https://github.com/proot-me/proot/issues/198).